Log/lambda_function.py

```diff
@@ -216,18 +216,11 @@ def is_cloudtrail(key):
     return bool(match)
 
 def parse_event_source(event, key):
-    if "lambda" in key:
-        return "lambda"
+    for source in ["lambda","elasticloadbalancing","redshift","cloudfront","kinesis", "mariadb","mysql","apigateway", "route53", "vpc", "rds"]:
```

Comment:
nit: you should install python linters, so we have a common way of formatting code. Usually we do `[x, y]` instead of `[x,y]`